scripts/build_test.py

```diff
@@ -243,6 +247,9 @@ def test_process_html(self):
         BASE_HTML_SOURCE_PATH = os.path.join(
             MOCK_TEMPLATES_DEV_DIR, 'base.html')
         BASE_JS_RELATIVE_PATH = os.path.join('pages', 'Base.js')
+        BASE_JS_FILE_URL = (
```

Comment:
Avoid logic in tests -- just define BASE_JS_FILE_URL = 'pages/Base.js'.

Reply:
Updated